docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/matrix/inverse-14.rst

```diff
@@ -25,21 +25,21 @@ where :math:`I` is the n-dimensional identity matrix.
 
 The inverse matrix exists if and only if the input matrix is invertible (satisfies any of the properties of the *Invertible Matrix Theorem*). In that case, the inverse exists, and is unique. However, if the matrix is not invertible, the operation may raise an exception or return an undefined result. The operation may return slightly different results for the same input data on different devices due to parallelism and different data types implementations.
 
-This operation can be used to compute the adjugate matrix instead of the inverse.
+This operation can be used to compute the adjoint matrix instead of the inverse.
```

Comment:
```suggestion
This operation can be used to compute the inverse of adjoint matrix.
```

Reply:
Please formulate more correctly because adjoint is just a transpose for real matrix. But when we say Inverse with adjoint=True it means to compute the inverse of input adjoint matrix.

Adjoint is not about inverse!

Reply:
Fixed!